templates/opt/graphite/conf/carbon.conf.erb

```diff
@@ -32,6 +32,10 @@
 #
 LOCAL_DATA_DIR = <%= scope.lookupvar('graphite::local_data_dir_REAL') %>/
 
+<% unless [:undef, nil].include? scope.lookupvar('graphite::gr_storage_dir') -%>
+PID_DIR = <%= scope.lookupvar('graphite::gr_storage_dir') %>
```

Comment:
This has to be scope.lookupvar('graphite::storage_dir_REAL')  to match the init scripts.